internal/dcache/cluster_manager/cluster_manager_impl.go

```diff
@@ -114,6 +128,81 @@ func (c *ClusterManagerImpl) ReportRVFull(rvName string) error {
 	return nil
 }
 
+func (cmi *ClusterManagerImpl) createClusterMapIfRequired(dCacheConfig *dcache.DCacheConfig, rvList []dcache.RawVolume) error {
```

Comment:
checkAndCreateInitialClusterMap() conveys the intent better. IfRequired could mean that it's dependent on some other conditions too, other than just the existence.